src/api/db/data/20190116105222_sanitize_reviews.rb

```diff
@@ -0,0 +1,36 @@
+class SanitizeReviews < ActiveRecord::Migration[5.2]
```

Comment:
so this here is probably to bring the reviews to a valid state, if this PR gets merge. This data migration isn't necessary for b.o.o and b.s.d but maybe some customer is using multiple review targets. Not sure if It will work as expected.

Reply:
I'm not really sure if we need this. I would drop it.